ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/JkSelf/velox.git
+VELOX_BRANCH=add-smj-operator
```

Comment:
Would you like to open this PR to support SMJ?

Reply:
Filed [PR#97](https://github.com/oap-project/velox/pull/97) to support SMJ in velox.

Reply:
Velox branch could be changed back, thanks.